terminus-ui/selection-list/src/selection-list.component.ts

```diff
@@ -653,7 +653,7 @@ export class TsSelectionListComponent implements
         const controlValue = this.ngControl.value;
         // If the input value doesn't match the selection, then the user must have edited the input value
         // istanbul ignore else
-        if (inputValue !== ((controlValue && controlValue.length > 0) ? this.displayFormatter(controlValue[0]) : '')) {
+        if (controlValue && inputValue !== ((controlValue && controlValue.length > 0) ? this.displayFormatter(controlValue[0]) : '')) {
```

Comment:
I think it's cause we always assume the control has a string value. I only noticed cause the demo had controls initialized to `null` so this broke as `cannot read length of null`. I'd hope TS would yell at people for this, but did this just in case.